gems/sorbet-runtime/lib/types/private/methods/decl_builder.rb

```diff
@@ -85,8 +85,8 @@ def checked(level)
       if !decl.checked.equal?(ARG_NOT_PROVIDED)
         raise BuilderError.new("You can't call .checked multiple times in a signature.")
       end
-      if !decl.soft_notify.equal?(ARG_NOT_PROVIDED)
-        raise BuilderError.new("You can't use .checked with .on_failure.")
+      if level == :never && !decl.soft_notify.equal?(ARG_NOT_PROVIDED)
```

Comment:
can you rename `soft_notify`. This is pretty confusing

Reply:
#1155